cpu/stm32l4/include/vendor/stm32l412xx.h

```diff
@@ -288,8 +288,6 @@ typedef struct
 
 /**
   * @brief External Interrupt/Event Controller
-  *
-  * Note: IMR1, EMR1, ... were changed to IMR, EMR, ...
```

Comment:
Please don't do this!
Change the code, not the vendor files. Things will break if a new version of the vendor file gets released or a new MCU version is added.

Reply:
Agreed with @benpicco on continuing like this for now. See #12163 for more info.